src/fu-engine.c

```diff
@@ -6915,8 +6929,14 @@ fu_engine_check_firmware_attributes(FuEngine *self, FuDevice *device)
 	subsystem = fu_udev_device_get_subsystem(FU_UDEV_DEVICE(device));
 	if (g_strcmp0(subsystem, "firmware-attributes") == 0) {
 		g_autoptr(GError) error = NULL;
-		if (!fu_context_reload_bios_settings(self->ctx, &error))
+		if (!fu_context_reload_bios_settings(self->ctx, &error)) {
 			g_debug("%s", error->message);
+			return;
+		}
+		if (!fu_engine_apply_default_bios_settings_policy(self, &error)) {
+			g_debug("%s", error->message);
```

Comment:
This I wonder if it should actually be `g_warning` so an administrator sees in their logs that some policies weren't loading properly.

Reply:
MSTM.